config/locales/en.yml

```diff
@@ -20,6 +20,7 @@ en:
       no_changes: "No changes made"
       account_updated: "Your account has been updated"
       none_selected: "Please select a user to edit"
+      inactive_user: "Success! The user's account has been marked as inactive"
```

Comment:
Just a thought, but it might be nice to include the name here, so you can verify which user was de-activated?